king_phisher/client/gui_utilities.py

```diff
@@ -513,6 +514,35 @@ def gtk_widget_destroy_children(widget):
 	for child in widget.get_children():
 		child.destroy()
 
+def set_listbox_classifiers(label, listbox, classifiers):
```

Comment:
You wrote the documentation but forgot to add it to the `docs/source/king_phisher/client/gui_utilities.rst` file for it to be included in the build.